gridsub_data.sh

```diff
@@ -83,6 +90,11 @@ for data_path in ${data_path_list[*]} ; do
 	cd $work/$job_name/
 	$work/$job_name/gridrun_data.sh $nevents $run_num $data_file | tee $work/$job_name/log/log.txt
 	cd -
-    fi
+    fi | tee $dir_macros/single_log_gridsub.txt
+   
+    JOBID="$(grep -o '\S*@jobsub\S*' <<< $(tail -2 $dir_macros/single_log_gridsub.txt | head -1))"
```

Comment:
JOBID="$(tail -2 $dir_macros/single_log_gridsub.txt | head -1 | grep -o '\S*@jobsub\S*')"

Reply:
Thank you, I have modified accordingly.